cmake/config/template-arguments.in

```diff
@@ -69,20 +69,18 @@ REAL_SERIAL_VECTORS := { Vector<double>;
                     @DEAL_II_EXPAND_PETSC_MPI_BLOCKVECTOR_REAL@;
                   }
 
-REAL_NONBLOCK_VECTORS := { Vector<double>;
-                    Vector<float> ;
-
-                    LinearAlgebra::Vector<double>;
-                    LinearAlgebra::Vector<float> ;
-
+REAL_NONBLOCK_VECTORS_MF := { 
```

Comment:
this is a list of vectors which should be enough to project from `Table<>` quadrature data in matrix-free.